src/test/java/org/ladocuploader/app/journeys/LaDigitalAssisterFlowJourneyTest.java

```diff
@@ -300,15 +252,51 @@ void fullDigitalAssisterFlow() {
 
     testPage.clickContinue();
     assertThat(testPage.getTitle()).isEqualTo("Criminal Justice");
+    testPage.clickButton("Yes");
 
-    testPage.navigateToFlowScreen("laDigitalAssister/householdVictimOfDomesticViolence");
-    assertThat(testPage.getTitle()).isEqualTo("Domestic Violence Victim");
+    assertThat(testPage.getTitle()).isEqualTo("Signpost");
+    testPage.clickContinue();
+    assertThat(testPage.getTitle()).isEqualTo("Job search");
+    testPage.clickButton("Yes");
+    assertThat(testPage.getTitle()).isEqualTo("Job search who");
+    testPage.clickElementById("jobSearch-you");
+    testPage.clickContinue();
+    assertThat(testPage.getTitle()).isEqualTo("Work disqualifications");
     testPage.clickButton("No");
-    assertThat(testPage.getTitle()).isEqualTo("Criminal Justice Involvement Warning");
 
+    // household income
+    testPage.navigateToFlowScreen("laDigitalAssister/householdIncomeByJob");
```

Comment:
Are we missing some employment screens before this?

Reply:
hi Sree - there were some screens missing.

Looks like Lauren first added home income and then added jobs individually so this tests both versions of the code.